database/src/migrations/release.0.7/create_spatial_extensions.psql

```diff
@@ -0,0 +1,8 @@
+-- create_spatial_extensions.psql

```

Comment:
Just double checking: the extension for this file is `.psql` and not `.sql`.  Is that correct?

Reply:
either way